portfolio/src/main/webapp/quiz.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Tal's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <ul>
+        <li><a class="active" href="index.html">Home</a></li>
+        <li><a href="hobbies.html">Hobbies</a></li>
+        <li><a href="dartmouth.html">School/Profession</a></li>
+        <li><a href="blog.html">Blog</a></li>
+        <li><a href="quiz.html">Quiz</a></li>
+
+    </ul>
+    <p> </p>
+    <div id="content">
+      <div id="wrapper">
+      <h1>Outdoor Activity Quiz</h1>
+      <p>Bored at home? Take this quiz to find out 
+          which outdoor adventure you should go on this summer
+          OR an activity you can look forward to after the pandemic!</p>
+      <form id="quiz">
```

Comment:
Reason the page is refreshing is because there's no form target here, and I believe it defaults to `target="_self"` if not specified, which would mean refresh self (ie. refresh the current page).

Try adding maybe either:
`<form id="quiz" target="#">`
or
`<form id="quiz" target="">`

Another option (or something you can do in conjunction if the above doesn't help), would be to change your button so it doesn't trigger a form submit.  Either:
`<button id="submit" onclick="tabulateAnswers()">Submit Your Answers</button>`
or
`<button type="submit" id="submit" onclick="tabulateAnswers(); return false;">Submit Your Answers</button>`